Source/Concurrency/InductiveSequentialization.cs

```diff
@@ -604,6 +819,17 @@ protected override List<Declaration> GenerateCheckers()
       {
         decls.AddRange(GenerateStepChecker(elim));
       }
+      if (rule == InductiveSequentializationRule.ISR){
+       decls.AddRange(GenerateSideConditionChecker(targetAction));
+       foreach (var elim in eliminatedActions)
+      {
+        if(elim == targetAction){
+          continue;
+        }
+        decls.AddRange(GenerateSideConditionChecker(elim));
+      }
+       decls.AddRange(DescendantCheckerFull());
```

Comment:
Use consistent naming for names of methods: GenerateSideConditionChecker and DescendantCheckerFull.